regolith/main_functions.go

```diff
@@ -47,6 +47,20 @@ func Install(filters []string, force, refreshResolvers, debug bool) error {
 	if err != nil {
 		return burrito.WrapError(err, "Unable to load config file.")
 	}
+	// Check if selected profiles exist
+	if add {
+		if len(profiles) == 0 {
+			profiles = []string{"default"}
+		}
+		// Get the profile
+		for _, profile := range profiles {
+			_, err := FindObjectByJSONPath(config, "regolith/profiles/"+profile)
```

Comment:
This way we removed an option of having `/` in the profile name but I guess it doesn't matter.

Reply:
Added escaping `/` in paths